content/raw/general/onboarding.md

```diff
@@ -0,0 +1,260 @@
+---
+title: "Login & Onboarding spec"
+description: ""
+date: 2022-08-16T08:48:57+00:00
+lastmod: 2022-08-16T08:48:57+00:00
+draft: false
+images: []
+menu:
+  raw:
+    parent: "general"
+weight: 100
+toc: true
+---
+
+# Login & Onboarding
+
+## Motivation
+
+## Definitions
+
+- Touch ID - Electronic fingerprint recognition used by apple desktop devices
+
+## Designs
+
+- [First time opening app](https://www.figma.com/file/17fc13UBFvInrLgNUKJJg5/Kuba%E2%8E%9CDesktop?node-id=1075%3A312335)
+- [Login](https://www.figma.com/file/17fc13UBFvInrLgNUKJJg5/Kuba%E2%8E%9CDesktop?node-id=1023%3A298171)
+- [New user](https://www.figma.com/file/17fc13UBFvInrLgNUKJJg5/Kuba%E2%8E%9CDesktop?node-id=1027%3A302661)
+
+## Use Cases
+
+## Functional Requirements
+
+### Initial Screen
+
+- If and only if it's the first time opening the app then the Initial Screen MUST BE displayed
+- The allow notifications screen MUST be displayed
+- The user MUST agree to the terms to proced
+    - `I aknowledge that Status Desktop is in Beta and by using it, I take the full responsibility for all risks concerning my data and funds.`
+    - Terms of Use
+    - Privacy Policy
+- The following two options should be shown
+    - "I'm new to Status"
+    - "I already use Status"
+
+### Login & Accounts
+
+#### Accounts and types of Logins
+
+- Each account MUST display:
+    - The profile picture
+        - If no profile picture is available then the initials matching the username MUST be displayed
+    - The Identicon ring
+        - If and only if the account uses an ENS name the identicon ring MUST NOT be displayed
+    - The Display name
+        - If and only if the account uses an ENS name, then the ENS name MUST BE displayed instead
+    - An icon repesenting the type of login
+        - If and only if the account uses a keycard login then the keycard icon MUST BE displayed
+        - If the account has a non keycard login then the icon MUST NOT be displayed
+- Each account MUST have a type of login:
```

Comment:
@iurimatias an account may log in in one of two possible ways:
1. using password - implies that for this device and this account keycard was not set up
2. using keycard (using pin) - implies that for this device and this account keycard was set up

for both of above mentioned ways user should be able to store to the keychain either password in case of the way `1` or pin in case of the way `2`.